pkg/steps/release/create_release.go

```diff
@@ -193,21 +193,38 @@ func (s *assembleReleaseStep) Run(ctx context.Context, dry bool) error {
 		return s.importFromReleaseImage(ctx, dry, providedImage)
 	}
 
-	stable, err := s.imageClient.ImageStreams(s.jobSpec.Namespace).Get(streamName, meta.GetOptions{})
-	if err != nil {
-		if errors.IsNotFound(err) {
+	var stable *imageapi.ImageStream
+	var cvo string
+	// waiting for importing the images
+	// 2~3 mins: build01 on aws imports images from api.ci on gci
+	log.Printf("waiting for importing cluster-version-operator ...")
+	if err := wait.Poll(10*time.Second, 5*time.Minute, func() (bool, error) {
+		stable, err = s.imageClient.ImageStreams(s.jobSpec.Namespace).Get(streamName, meta.GetOptions{})
+		if err != nil {
+			return false, err
+		}
+		pullSpec, ok := resolvePullSpec(stable, "cluster-version-operator", true)
+		if !ok {
+			return false, nil
+		}
+		log.Printf("Found pull spec for cluster-version-operator")
+		cvo = pullSpec
+		return true, nil
+	}); err != nil {
+		if wait.ErrWaitTimeout == err {
+			log.Printf("No %s release image necessary, %s image stream does not include a cluster-version-operator image", tag, streamName)
+			return nil
+		} else if errors.IsNotFound(err) {
 			// if a user sets IMAGE_FORMAT=... we skip importing the image stream contents, which prevents us from
 			// generating a release image.
 			log.Printf("No %s release image can be generated when the %s image stream was skipped", tag, streamName)
 			return nil
 		}
 		return fmt.Errorf("could not resolve imagestream %s: %v", streamName, err)
 	}
-	cvo, ok := resolvePullSpec(stable, "cluster-version-operator", true)
-	if !ok {
-		log.Printf("No %s release image necessary, %s image stream does not include a cluster-version-operator image", tag, streamName)
-		return nil
-	}
+	// This should be put into wait too
+	// In the tests, "cli" tag has been imported already when the above is done
```

Comment:
Just do it now, one `wait.Poll` that returns `true` only when both have finished

Reply:
Done.